### ISSUE TITLE
[Settings]: When tooltip is visible on the screen Voice Access are not able to access any control on the Terminal app.

### ISSUE THREAD

**Issue by v-rpundir** [microsoft/terminal#13822]
### Windows Terminal version

1.15.2003.0

### Windows build number

10.0.25179.1000

### Other Software

**Test Environment:**
OS: Window 11 22H2 OS Build 25179.1000
App: Windows Terminal Preview
AT: Voice Access

### Steps to reproduce

**Repro Steps:**
1. Open Windows Terminal.
2. Open Voice Access AT.
3. Inside Settings>Startup page, Use 'Expand launch Size' command, when command get executed, a tooltip will appear. 
4. When Tooltip is visible on screen, try to access app using available commands without dismissing the tooltip.

**User Experience:**
People with limited mobility who use Voice Access will impacted here as they might not be able to use controls and get confused regarding the same.

**Guideline Reference:**
EN 301 549 V3.2.1: 11.06.02 No disruption of accessibility features

**Attachments:**
[Tooltip.zip](https://github.com/microsoft/terminal/files/9400232/Tooltip.zip)

Attachment Note: Not able to inspect Terminal Preview app with AI and Inspector tool.

### Expected Behavior

When tooltip is visible on the screen Voice Access users should be able to access all control on the Terminal app.

### Actual Behavior

When tooltip is visible on the screen Voice Access are not able to access any control on the Terminal app like nothing happens when users use 'Click control name' and 'show numbers' etc. commands.

**Observation:**
Voice Access users can access controls after dismissing the tooltip or using 'Show Grid' command.

To learn more about Voice Access and how an application can support its different modes, Refer https://www.osgwiki.com/wiki/VoiceAccess_Testing

Voice Access Commands page: https://sway.office.com/Q5JvoIGBI5glyoKw?ref=Link

**Comment by zadjii-msft** [comment]
This definitely isn't specific to one single control - the mouse was left on top of the "launch size" expando, and that caused the tooltip to appear over it. 

Now, the bigger question here is how should Voice access interact with XAML tooltips? Clearly it can't walk around the tooltips in the UI tree. I suspect this needs to be moved way upstream, either into WinUI itself, or into Voice Access. @carlos-zamora thoughts?

**Comment by carlos-zamora** [comment]
Yup, this is a XAML Islands bug. I've filed https://github.com/microsoft/microsoft-ui-xaml/issues/7815 on the XAML team

CC @v-rpundir

**Comment by v-rpundir** [comment]
Tracking ID: [Bug 41849571](https://microsoft.visualstudio.com/DefaultCollection/OS/_workitems/edit/41849571): [XAML Controls Gallery -> ToolTip]: When tooltip is visible on screen, unable to access app using available commands ('Click control name' and 'show numbers' etc.) without dismissing the tooltip.

FYI- We do not track XAML bug in GitHub as per XAML testing team guidelines.

**Comment by damian-666** [comment]
hey, Mr  Carlos,  just an idea.  Just from kind of having a quick look over this vault, it seems like you're you're pushing this forward more so than anyone else. So I want to. Just run it right to you. Another protocol on this. I see
 you work for Microsoft, and this isn't really an open source project. And I just. wanted to throw ideas at you. I'm a user. I'll use whatever works for me. I've been using windows a 99% my career. I don't want to use Ubuntu in Julia, but I think I have to for this one's the scientific project which could have a huge impact, but I just noticed in power tools, that there is a OCR feature that Google's always had, but which will let you maybe go to a video game which has buttons rendered
 
  and get the text from those buttons and then allow you to press buttons and then you won't have to do all the work that you've done with all these different UI protocols and stuff. What I would think you would do for XAML or for anything if you get into that visual tree somehow I don't even know how to do it is you find a button via name and localization.  Have the words that you want. to say put those into a dictionary and limit the language models domain so that it will guess first that you might be referring to those buttons if you're in that context. before just you know, putting up numbers or letters I would suggest that I'm just going to look at my screen if I see a button and I'm going to say its name. If I see a tool tip by my I might remember what was on that tool tip and use in that the next time I want to invoke the command I might not even be looking at my screen after tat . know, so I really think some of those work that's being done is redundant. And maybe you just do the ocr. I know it's a little bit more expensive, but when you're feeding it into the voice to text AI  first. it's going to make that cheaper, more accurate,  and it's going to get implemented faster and the faster it is out there and better , you're going to save the planet 20,000 hours of mucking around clicking and mousing

**Comment by carlos-zamora** [comment]
Hi @damian-666, I suggest you submit this feedback to the relevant team (sounds like Voice Access, maybe) via Feedback Hub. Here's a link: https://support.microsoft.com/en-us/windows/send-feedback-to-microsoft-with-the-feedback-hub-app-f59187f8-8739-22d6-ba93-f66612949332

That'll make sure it gets to the right place :)
